tests/Boiler2Tests.cs

```diff
@@ -66,7 +66,7 @@ public void TemperatureRisesAndFallsHeaterToggles()
         currentTemperatureDegrees.Should().Be(101f);
         heaterState.Should().BeTrue();
 
-        // Temperature rises until 123�, then falls with heater off, step -5�.
+        // Temperature rises until 123°, then falls with heater off, step -5°.

```

Comment:
I see strange chars here as well

Reply:
Not sure, what happened here but now it looks correct